server/db.mjs

```diff
@@ -56,20 +56,40 @@ export const deleteTask = async (id) => {
     [id],
   );
 
-  console.log({ deletedTask });
+  window.alert("Deleted Task: ", { deletedTask });
```

Comment:
Why did you change this from `console.log()` to `window.alert()`?

Reply:
Playing with the idea of a notifications system. So my thought process:  windows.alert() feels like a “sudo” notification so I thought I’d implement one to simulate and see how it feels..?

Reply:
Gotcha! I just was curious if it still shows up if the command is on the server side 😅 very neat if it does!!